### DISCUSSION TITLE
核心依赖的问题：iptables-mod-conntrack-extra

### DISCUSSION THREAD

**Discussion by ghwwx** [immortalwrt/immortalwrt#1437] [General]
有一个FROS的插件，原作者用Immortalwrt的源代码加上自己的插件编译了一个固件。然后，我下载了这个固件，然后我再往这个固件安装passwall插件的时时候，会告诉我有两个插件没有安装，我手动安装时，告诉我核心hash值不一致导致不能安装。强制安装时，一个插件没有问题。
但是当我强制安装iptables-mod-conntrack-extra插件时，机器就会重启，并且luci也无法访问。

不知道如何解决。

**Comment by xlighting2017** [comment]
有一个 番茄花园 的 windows改版， 我在用这个改版，然后再往这个改版里面安装 AutoCAD，会提示 DLL错误，我强制覆盖了这2个DLL， 电脑就蓝屏死机了

这时候，我该找微软给我解决么？
